### PR TITLE
Add some details about certs

### PR DESCRIPTION
**Pull request by MalloZup**
Adress UX https://github.com/kubernetes/kubeadm/issues/1361

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -23,9 +23,12 @@ You should be familiar with [PKI certificates and requirements in Kubernetes](/d
 
 ## Renew certificates with the certificates API
 
-Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands.
+The Kubernetes certificates reach normally their expiration date after one year.
```
Comment (zparnold) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]:
```suggestion
The Kubernetes certificates normally reach their expiration date after one year. 
```

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 2]
```diff
@@ -23,9 +23,12 @@ You should be familiar with [PKI certificates and requirements in Kubernetes](/d
 
 ## Renew certificates with the certificates API
 
-Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands.
+The Kubernetes certificates reach normally their expiration date after one year. 
+
+Kubeadm can renew certificates with the `kubeadm alpha certs renew` commands; you should run these commands on Master only.
```
Comment (zparnold) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: @MalloZup Can you provide the reason why they should be done on the master?

Comment (neolit123) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: sadly our procedure for rotation is quite undocumented.
i can confirm that this has to be done on control plane nodes only.

with regards to your PR @MalloZup 

please replace:
`you should run these commands on Master only`
with:
`you should run these commands on control-plane nodes only`

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 3]
```diff
@@ -23,9 +23,12 @@ You should be familiar with [PKI certificates and requirements in Kubernetes](/d
...
 
 Typically this is done by loading on-disk CA certificates and keys and using them to issue new certificates.
+
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md]: is this whitespace change needed?
please remove, if not.

**Comment by MalloZup** [comment]
th x @neolit123 @zparnold i will adress your comments soon

**Comment by MalloZup** [comment]
@neolit123 @zparnold  please review once you have time. TIA :rocket:

**Comment by Rajakavitha1** [comment]
/assign @Bradamant3 
Could you please take a look at this PR.

**Pull request merged**